src/docs/07-operation-guide/01-setup.md

```diff
@@ -0,0 +1,348 @@
+| services.NAME.pprof | Golang profiling service , will bind on the same IP as RPC | a port that you want to serve pprof request |
+| services.Name.metrics | See Metrics&Logging section | cc |
+| clusterMetadata | Cadence cluster configuration. <br/><br/>enableGlobalDomain：true will enable Cadence Cross datacenter replication(aka XDC) feature.<br/><br/>failoverVersionIncrement: This decides the maximum clusters that you will have replicated to each other at the same time. For example 10 is sufficient for most cases.<br/><br/>masterClusterName: a master cluster must be one of the enabled clusters, usually the very first cluster to start. It is only meaningful for internal purposes.<br/><br/>currentClusterName: current cluster name using this config file. <br/><br/>clusterInformation is a map from clusterName to the cluster configure <br/><br/>initialFailoverVersion: each cluster must use a different value from 0 to failoverVersionIncrement-1. <br/><br/>rpcName: must be “cadence-frontend”. Can be improved in this issue. <br/><br/>rpcAddress: the address to talk to the Frontend of the cluster for inter-cluster replication. <br/><br/>Note that even if you don’t need XDC replication right now, if you want to migrate data stores in the future, you should enable xdc from every beginning. You just need to use the same name of cluster for both masterClusterName and  currentClusterName.  See more details in Migration section. | As explanation. |
+| dcRedirectionPolicy | For allowing forwarding frontend requests from passive cluster to active clusters.  | “selected-apis-forwarding” |
+| archival | This is for archival history feature, skip if you don’t need it. See more in History Archival section | N/A |
+| blobstore | This is also for archival history feature Default cadence server is using file based blob store implementation.  | N/A |
+| domainDefaults | default config for each domain. Right now only being used for Archival feature.  | N/A |
+| dynamicConfigClient | Dynamic config is a config manager that you can change config without restarting servers. It’s a good way for Cadence to keep high availability and make things easy to configure. <br/><br/>Default cadence server is using FileBasedClientConfig. But you can implement the dynamic config interface if you have a better way to manage.| Same as the sample development config |
+| persistence | Configuration for data store / persistence layer. <br/><br/>Values of DefaultStore VisibilityStore AdvancedVisibilityStore should be keys of map DataStores. <br/><br/>DefaultStore is for core Cadence functionality. <br/><br/>VisibilityStore is for basic visibility feature <br/><br/>AdvancedVisibilityStore is for advanced visibility<br/><br/> See [persistence documentation](https://github.com/uber/cadence/blob/master/docs/persistence.md) about using different database for Cadence| As explanation |
+
+
+### Dynamic configuration overview
```

Comment:
It might be worth splitting Static configs, Dynamic configuration overview,  Dynamic Configuration for Frontend Service, Dynamic Configuration for Matching Service, etc... into separate pages. That way if users want to jump around or search for a page it will be abit easier.

Reply:
I tried that it looks pretty bad because the name is too long in a side bar. I think it's good enough to locating at dynamic configuration level. Usually I think people would just CTRL+F to search for things that they need in the table

Reply:
Ok no problem at all